docs/components/board/_index.md

```diff
@@ -0,0 +1,1534 @@
+
+**Returns:**
+
+- `reader` [(AnalogReader)](https://pkg.go.dev/go.viam.com/rdk/components/board#AnalogReader): An interface representing an analog reader configured and residing on the board.
+- `ok` [(bool)](https://pkg.go.dev/builtin#bool): True if there was an analog reader of this `name` found on your board.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/board#Board).
+
+```go
+myBoard, err := board.FromRobot(robot, "my_board")
+
+// Get the AnalogReader "my_example_analog_reader".
+reader, err := myBoard.AnalogReaderByName("my_example_analog_reader")
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### DigitalInterruptByName
+
+Get an [`DigitalInterrupt`](#digital_interrupts) by `name.`
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `name` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Name of the digital interrupt you want to retrieve. Set as the `"name"` property [in configuration](/components/board/#digital_interrupts).
+
+**Returns:**
+
+- `interrupt` [(DigitalInterrupt)](https://python.viam.dev/_modules/viam/components/board/board.html#Board.DigitalInterrupt): An interface representing a configured interrupt on the board.
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/board/index.html#viam.components.board.Board.digital_interrupt_by_name).
+
+```python
+my_board = Board.from_robot(robot=robot, name=)
+
+# Get the DigitalInterrupt "my_example_digital_interrupt".
+interrupt = await my_board.digital_interrupt_by_name(name="my_example_digital_interrupt")
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `name` [(string)](https://pkg.go.dev/builtin#string): Name of the digital interrupt you want to retrieve. Set as the `"name"` property [in configuration](/components/board/#digital_interrupts).
+
+**Returns:**
+
+- `interrupt` [(DigitalInterrupt)](https://pkg.go.dev/go.viam.com/rdk/components/board#DigitalInterrupt): An interface representing a configured interrupt on the board.
+- `ok` [(bool)](https://pkg.go.dev/builtin#bool): True if there was a digital interrupt of this `name` found on your board.
+
+For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/components/board#Board).
+
+```go
+myBoard, err := board.FromRobot(robot, "my_board")
+
+// Get the DigitalInterrupt "my_example_digital_interrupt".
+interrupt, ok := myBoard.DigitalInterruptByName("my_example_digital_interrupt")
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GPIOPinByName
+
+Get a `GPIOPin` by its pin number.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `name` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): Pin number (NOT GPIO number) of the GPIO pin you want to retrieve.
+Refer to the pinout diagram and data sheet of your [board model](#configuration) for pin numbers and orientation.
```

Comment:
```suggestion
Refer to the pinout diagram and data sheet of your board for pin numbers and orientation.
```
This would be for the board itself, not for the Viam model

Reply:
But this leads to supported board models & links in case they are confused and it would have to be a model? I don't think it hurts

Reply:
I don't want them to think that this link contains the relevant info since it doesn't contain a data sheet or pinout